samples/attested_tls/README.md

```diff
@@ -93,3 +93,17 @@ mkdir build && cd build
 cmake ..
 make run
 ```
+### Running attested tls sample on Kubernetes
+```bash
+make CLOUD_DEPLOYMENT=true
+make run
+```
+`CLOUD_DEPLOYMENT` flag modifies behavior of the test to make it
+suitable for scenario where the server is deployed in the form of a replicated set and is accessed via a load balancer.
+An example of this, would be as a Service on a Kubernetes cluster.
```

Comment:
Fix grammar
```suggestion
The `CLOUD_DEPLOYMENT` flag modifies the behavior of the test to make it
suitable for the scenario where the server is deployed in the form of a replicated set and is accessed via a load balancer.
An example of this would be as a Service on a Kubernetes cluster.
```